### ISSUE TITLE
Blink scripts will no longer work as of 5/11/2020 ( It is now fixed! )

### ISSUE THREAD

**Issue by MattChappel** [nayrk/Blink#5]
Just got this email.   This looks bad... real bad.   : - ( 
Have you heard anything about this?

-----------------------------
Dear Blink Customer,

You are receiving this email because your account has used automation scripts that use an outdated login method to connect to your Blink account. With security and customer protection as our top priority, that login method is being deprecated.  This means that any scripts you are using will no longer work as of 5/11.
 
We encourage you to try out IFTTT, our supported 3rd party integration, for more secure automation functionality. If you have other feature preferences or requests, you can always email product feedback to info@blinkforhome.com for consideration.
 
Thank you,
 
Team Blink

**Comment by MattChappel** [comment]
yep sure enough... As of today (5/11/2020) this awesome script does not work, because blink has changed their security.   I doubt nayrk can fix this.   It was great while it lasted.

**Comment by m-holler** [comment]
Please Update your Script, you just have to add the verification code. I use a other script for homeautomation, and the autor has add the field for verfication code, and now it work´s again with blink. I hope you can add this in your awesome Powershellscript soon.

**Comment by colinreid89** [comment]
I have updated the code and the download works again. Enjoy ;)

[BlinkVideoDownloaderGeneral.zip](https://github.com/nayrk/Blink/files/4637416/BlinkVideoDownloaderGeneral.zip)

**Comment by nayrk** [comment]
> I have updated the code and the download works again. Enjoy ;)
> 
> [BlinkVideoDownloaderGeneral.zip](https://github.com/nayrk/Blink/files/4637416/BlinkVideoDownloaderGeneral.zip)

Appreciate it. Been too busy with work to fix this.

**Comment by sutty550e** [comment]
Thanks Colin. I've updated mine, in accordance with your version, and it's working again for me now. Good stuff, and thanks again.

**Comment by MattChappel** [comment]
yep thanks Thanks Colin.  works great again.   Thanks.

**Comment by amitdraja** [comment]
Thanks a lot colinreid89 and nayrk  !!!!

**Comment by hujiaxin1029** [comment]
Hi all,

I tried the code and it says the video has already been downloaded, but it actually didn;t. 
Can anyone help?

Thanks,
Mark

**Comment by sutty550e** [comment]
HI Mark

I think Ryan is extremely busy with work, as a result of Covid-19, so he may not have time to drop in and help. Maybe others can assist, but we may need a little more information as to what you've tried, where you've looked for it, what you might have amended in the script, etc? Maybe even some screen shots of the script, perhaps as it executes and fails, might be helpful. If you copy your script back here, for advice, be sure to edit out your personal information first.

Hope someone can help.

By the way, you have posted in a closed issue, so maybe it won't be seen as readily as if it were new?

Regards

Andrew

**Comment by hujiaxin1029** [comment]
Thanks for the reply Andrew, I have uploaded some details in a new opened issue. Thanks!

**Comment by TeagenSirrus** [comment]
You guys are amazing. I was so angry when the program stopped working, but leave it to your amazing minds to get something up and running in almost no time.

**Comment by BroomofBronx** [comment]
I have noticed that the Blink Minis cameraclips doesn´t download. I haven't done much programing in my days, but a little for a long time ago. I managed to add Write-Host, now I can see that the code finds all of my three networkNames CMK, CMK2 and CMK3. In two of them the code finds all the cameras and downloads the clips, flawless. ;)
BUT the Blink Minis networkName doesn't get created as a folder on my harddrive and the code doesn't find the camera (the Blink Mini itself). Does the Mini need some other code?
Would be nice if someone could help me. Thanks.
![ps](https://user-images.githubusercontent.com/67604641/86287769-17d79580-bbe9-11ea-8e11-e9c789579739.png)
